.travis.yml

```diff
@@ -34,6 +34,52 @@ jobs:
           install:
               - travis_retry composer update --classmap-authoritative --no-suggest  --prefer-dist --prefer-lowest --prefer-stable
 
+        - &Database
+          stage: Test
+          env: DB=sqlite
+          php: 7.2
+          script:
+              - vendor/bin/phpunit --group database
+
+        - &MySQL
+          << : *Database
+          env: DB=mysql:5.7
+          sudo: true
+          services:
+              - docker
+          install:
+              - *Composer
+              - travis_retry docker pull ${DB}
+          before_script:
+              - .travis/run.sh "mysql" "mysqladmin ping --silent" "--env MYSQL_ALLOW_EMPTY_PASSWORD=yes --env MYSQL_DATABASE=test --publish 9000:3306 ${DB} ${OPTIONS}"
+              - export DB_URI=mysql://root@127.0.0.1:9000/test
+
+        - << : *MySQL
+          env: DB=mysql:8 OPTIONS=--default-authentication-plugin=mysql_native_password
+
+        - &MariaDB
+          << : *MySQL
+          env: DB=mariadb:5
+
+        - << : *MariaDB
+          env: DB=mariadb:10
+
+        - &PostgreSQL
+          <<: *MySQL
+          env: DB=postgres:9.2
```

Comment:
as much as people as more conservative about databases, we support PHP 7.2+ only (2017) but Postgres 9.2 (2012) which is [not officially supported itself](https://www.postgresql.org/support/versioning/); given the long build, I suggest pruning this older database version